src-electron/sdk/matter.js

```diff
@@ -92,17 +73,23 @@ async function computeStoragePolicyForGlobalAttributes(
     attributes.map(async (attribute) => {
       if (attribute.clusterId == null) {
         forcedExternal = await getForcedExternalStorage(db, attribute.id)
-        if (forcedExternal.byName?.[clusterName]?.includes(attribute.name)) {
-          attribute.storagePolicy =
-            dbEnum.storagePolicy.attributeAccessInterface
-        }
+        forcedExternal.map((option) => {
+          if (
+            option.optionCategory == clusterName &&
+            option.optionLabel == attribute.name
+          ) {
+            attribute.storagePolicy =
+              dbEnum.storagePolicy.attributeAccessInterface
+          }
+        })
       }
       return attribute
     })
   )
 }
-async function getDisabledStorage(db, zcl) {
-  return await getForcedExternalStorageList(db, zcl)
+
+async function getDisabledStorage(db) {
+  return getForcedExternalStorage(db)
```

Comment:
Whatever is calling getDisabledStorage can it just call  getForcedExternalStorage ?